internal-link-shortcode.php

```diff
@@ -75,18 +75,19 @@ function cfplsc_request_handler() {
 function cfplsc_id_lookup() {
 	global $wpdb;
 	$title = stripslashes($_GET['post_title']);
-	$wild = '%'.$wpdb->escape($title).'%';
-	$posts = $wpdb->get_results("
+	// $wild = '%'.$wpdb->prepare($title).'%';
+	$posts = $wpdb->get_results(
```

Comment:
This is a good change, but it should be in its own separate PR attached to its own separate issue